protos/vehicle.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+package vehicle;
+
+message Data {
+  enum Status {
+    PENDING = 0;
+    PROCESSING = 1;
+    FINISHED = 2;
```

Comment:
Is this changing in the next PR you're making? The SETTLEMENT, DATA_COLLECTION, AUCTION, RESOLVED?

Reply:
Correct